ontology/yaml/resources/HVAC/entity_types/SDC.yaml

```diff
@@ -92,4 +92,23 @@ SDC_UDEC:
   is_canonical: true
   implements:
   - SDC
-  - UDEC
\ No newline at end of file
+  - UDEC
+
+SDC_EXT_TLT_NORTH:
+  guid: "bb69d3c2-5063-425c-90c4-636f31d03a57"
+  description: "Directional shade (North)."
+  is_canonical: true
+  implements:
+  - SDC_EXT_TLT
+  uses:
```

Comment:
Can you please make a note in the description that illuminance is not required because in the northern hemisphere there is minimal heat gain from solar loads

Reply:
done